src/Language/PureScript/Docs/RenderedCode/RenderType.hs

```diff
@@ -154,12 +154,6 @@ insertPlaceholders :: RenderTypeOptions -> Type -> Type
 insertPlaceholders opts =
   everywhereOnTypesTopDown convertForAlls . everywhereOnTypes (convert opts)
 
-dePrim :: Type -> Type
-dePrim ty@(TypeConstructor (Qualified _ name))
-  | ty == tyBoolean || ty == tyNumber || ty == tyString =
-    TypeConstructor $ Qualified Nothing name
-dePrim other = other
```

Comment:
Do we not want to remove `Prim.` qualification anymore? Or is this handled somewhere else that I didn't notice now?